03_launch_mgmt_cluster.sh

```diff
@@ -219,6 +219,12 @@ if [[ "${PROVISIONING_IPV6}" == "true" ]]; then
   sudo su -l -c 'minikube ssh "sudo ip -6 addr add '"$CLUSTER_PROVISIONING_IP/$PROVISIONING_CIDR"' dev eth2"' "${USER}"
 else
   sudo su -l -c "minikube ssh sudo ip addr add $CLUSTER_PROVISIONING_IP/$PROVISIONING_CIDR dev eth2" "${USER}"
+  #sudo su -l -c 'minikube ssh sudo ip addr add 172.22.0.2/24 dev eth2' "${USER}"
+  sudo su -l -c 'minikube ssh sudo brctl addbr ironicendpoint' "${USER}"
+  sudo su -l -c 'minikube ssh sudo ip link set ironicendpoint up' "${USER}"
+  sudo su -l -c 'minikube ssh sudo brctl addif ironicendpoint eth2' "${USER}"
+  sudo su -l -c 'minikube ssh sudo ip addr add 172.22.0.6/$PROVISIONING_CIDR dev ironicendpoint' "${USER}"
+
```

Comment:
this will not survive a reboot. Should we make it so that it can ?

Reply:
Good point. Future task may be